File: Practice(Beginner)/C++/Two VS Ten.cpp
// Source: https://www.codechef.com/problems/TWOVSTEN

#include <bits/stdc++.h>
using namespace std;

int twoVsTen(int n){
    int i = -1;
    if(n%10 == 0){
        return 0;
    } else if (n%10 == 5){
        return 1;
    } else{
        return i;
    }
}

int main() {
    int t, n;
    cin >> t;
    while (t--){
        cin >> n;
        cout << twoVsTen(n) << endl;
    }
    return 0;
}

File: Practice(Beginner)/Python3/Two_VS_Ten.py
# Source: https://www.codechef.com/problems/TWOVSTEN

def twoVsTen(n):
    if (n % 10 == 0):
        return 0
    elif (n % 10 == 5):
        return 1
    else:
        return -1
t = int(input())
for test in range(t):
    n = int(input())
    print(twoVsTen(n))
